### PR TITLE
blog: devxconf 2022 wrap

### PR DESCRIPTION
**Pull request by ghuntley**
## Description

A blog post that wraps up the DevX Conf and shares updates on how funding is being distributed.

## Release Notes

```release-note
NONE
```

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2236"></a>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2236"></a>

### CODE REVIEW

File: src/routes/blog/devxconf-2022-wrap.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+Several of the hottest, most impactful developer tools of the current time were presented at DevX Conf. The conference is evolving to be a space where developer tool founders share what they've been working on all year. The themes that stood out in 2022 were automation (think, [Fig](https://www.youtube.com/watch?v=VhxGX8MV3Vw), [Warp](https://www.youtube.com/watch?v=EFbicktk6C4), [Supabase](https://www.youtube.com/watch?v=1Tz0FBoqCTQ)), collaboration (think, [Zed](https://www.youtube.com/watch?v=wXT73bBr83s)), and integration testing (think, [AtomicJar](https://www.youtube.com/watch?v=A_I8_3UFHo0)). The continued transition to cloud development was a centrepiece of discussions at DevX Conf. Next to Gitpod, we saw Okteto and [Codespaces](https://www.gitpod.io/blog/cake) talk about remote developer environments.
+
+[Last year](https://www.gitpod.io/blog/devxconf-wrap) was the first year of the DevX Conference, and we saw <b>TBA: attendees</b>. Over $10,000 USD was donated to unpaid volunteers and the maintainers of Open-source projects that underpin our digital infrastructure.
```
Comment (ghuntley) [inline src/routes/blog/devxconf-2022-wrap.md:39]: 🎗 correct this line before merging `<b>TBA: attendees</b>`

**Review by Cahllagerfeld — APPROVED** [review]
From the techincal side it looks good to me 🚀

**Comment by MichaelAring** [comment]
**Title**
Current: 

- "DevX Conf wrap and distributing USD $22,600 to Open-source projects"

Ideas:

- "Gitpod raises $22,600 at DevX Conf to give to Open-source maintainers"
- "We raised $22,600 at DevX Conf to give to Open-source projects you voted for"
- "Why we are giving all of the sponsor money from DevX Conf to Open-source projects"
- "Giving all of the sponsor money from DevX Conf to Open-source projects"
- "Giving $22,600 raised from DevX Conf sponsors to Open-source projects"
- "Based on your votes, we are distributing over $22,000 to Open-source projects"

File: src/routes/blog/devxconf-2022-wrap.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+</div>
+
+Only a fraction of funds has been distributed at this stage because, as an industry, we haven’t documented and established straightforward ways where companies can give Open-source projects money. If you are the steward of one of the above projects, please go sign up to GitHub Sponsors or Open Collective and/or publically document the process of how companies can support your project.
+
+At Gitpod, we often wonder what the future would look like if these high achievers that our digital society is built upon were empowered to become independent artists. If just one of those people can help more people better understand a technology or improve the developer experience for an entire ecosystem what is the worth/value of that and why isn’t our industry doing that yet? In business terms, Open-source maintainers are unpaid and unsecured vendors.
```
Comment (MichaelAring) [inline src/routes/blog/devxconf-2022-wrap.md:54]: Our way of speaking should be more solution-oriented, a bit less problem-oriented :)

I suggest this narrative:

- We acknowledge the importance of these projects
- We are aware they are underfunded and there is a lack of suitable reward mechanisms in the industry
- Therefore, we want to contribute by raising money at DevX Conf and giving it back to the community
- We know it is just a small step
- We will continue to this (and more of it) in the future

**Review by MichaelAring — CHANGES_REQUESTED** [review]
Comments and changes

See also my comment in Slack on potentially splitting the article

File: src/routes/blog/devxconf-2022-wrap.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
+---
+author: ghuntley
+date: Tuesday, 14 June 2022 16:00:00 UTC
+excerpt: The talks from DevX Conf are now available for viewing and USD $22,600 is being distributed to Open-source projects that conference attendees voted for.
+image: teaser.png
+slug: devxconf-2022-wrap
+subtitle: All twenty seven talks from the conference are now available on YouTube.
+teaserImage: teaser.png
+title: Gitpod raises $22,600 at DevX Conf to give to Open-source maintainers
```
Comment (Siddhant-K-code) [inline src/routes/blog/devxconf-2022-wrap.md:9]: This title looks good 🎉

**Review by Siddhant-K-code — APPROVED** [review]

File: src/routes/blog/devxconf-2022-wrap.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+---
+
+<script context="module">
+  export const prerender = true;x
```
Comment (Siddhant-K-code) [inline src/routes/blog/devxconf-2022-wrap.md:13]:
```suggestion
  export const prerender = true;
```

nit fix ReferenceError

**Review by Siddhant-K-code — CHANGES_REQUESTED** [review]

**Comment by MichaelAring** [comment]
With these latest changes reviewed and approved, I think we are ready to merge @ghuntley 👍

File: src/routes/blog/devxconf-2022-wrap.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
+---
+author: ghuntley
```
[the remark this answers is not in the document]
Comment (Siddhant-K-code) [reply src/routes/blog/devxconf-2022-wrap.md:2]: @MichaelAring i think it should be this: 

```suggestion
author: ghuntley, MichaelAring
```

**Pull request merged**